pie-dao/pools.piedao.org
# Feature/pie 482

##### Description
How to test:
from staking page, scroll down to the "rewards" component, click on your reward and check in the breakdown page if numbers/votes looks correct.

##### Checklist


- [ ] Test coverage hasn't decreased

##### Testing


##### Refers/Fixes
